src/Endpoints/class-graphql-metadata.php

```diff
@@ -0,0 +1,266 @@
+			'fields'      => array(
+				'articleSection'   => array( 'type' => 'String' ),
+				'author'           => array( 'type' => array( 'list_of' => self::GRAPHQL_AUTHOR_TYPE ) ),
+				'context'          => array( 'type' => 'String' ),
+				'creator'          => array( 'type' => 'String' ),
+				'headline'         => array( 'type' => 'String' ),
+				'image'            => array( 'type' => self::GRAPHQL_IMAGE_TYPE ),
+				'keywords'         => array( 'type' => array( 'list_of' => 'String' ) ),
+				'mainEntityOfPage' => array( 'type' => self::GRAPHQL_MAIN_ENTITY_TYPE ),
+				'publisher'        => array( 'type' => self::GRAPHQL_PUBLISHER_TYPE ),
+				'thumbnailUrl'     => array( 'type' => 'String' ),
+				'type'             => array( 'type' => 'String' ),
+				'url'              => array( 'type' => 'String' ),
+			),
+		);
+		/**
+		 * Filters the array for the custom type to represent metadata on GraphQL.
+		 *
+		 * @see https://wpgraphqldocs.gatsbyjs.io/functions/register_graphql_object_type/
+		 * @since 3.2.0
+		 *
+		 * @param array $meta_type Array with the fields of the new type.
+		 */
+		$meta_type = apply_filters( 'wp_parsely_graphql_meta_type', $meta_type );
+		register_graphql_object_type( self::GRAPHQL_META_TYPE, $meta_type );
+
+		$container_type = array(
+			'description' => __( 'Parse.ly Metadata root type.', 'wp-parsely' ),
+			'fields'      => array(
+				'version'  => array(
+					'type'        => 'String',
+					'description' => __( 'Revision of the metadata format.', 'wp-parsely' ),
+				),
+				'meta'     => array(
+					'type'        => 'ParselyMeta',
+					'description' => __( 'Structured and filterable metadata.', 'wp-parsely' ),
+				),
+				'rendered' => array(
+					'type'        => 'String',
+					'description' => __( 'HTML string containing the metadata. Intended to be rendered in the front-end as is.', 'wp-parsely' ),
+				),
+			),
+		);
+		register_graphql_object_type( self::GRAPHQL_CONTAINER_TYPE, $container_type );
+	}
+
+	/**
+	 * Register the custom metadata fields so they can be queried in GraphQL.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @return void
+	 */
+	private function register_fields(): void {
+		$options      = $this->parsely->get_options();
+		$object_types = array_unique( array_merge( $options['track_post_types'], $options['track_page_types'] ) );
+
+		/**
+		 * Filters the list of post object types that the Parse.ly GraphQL API is hooked into.
+		 *
+		 * @since 3.2.0
+		 *
+		 * @param string[] $object_types Array of strings containing the object types, i.e. `page`, `post`, `term`.
+		 */
+		$object_types = apply_filters( 'wp_parsely_graphql_object_types', $object_types );
+
+		foreach ( $object_types as $object_type ) {
+			$post_type_object = get_post_type_object( $object_type );
+
+			register_graphql_field(
+				$post_type_object->graphql_single_name,
+				self::FIELD_NAME,
+				array(
```

Comment:
Rather than such a large array making the end of the `register_graphql_field()` call difficult to find, extract this array into a variable, and make the `register_graphql_field()` a single line.

This will also help with reducing the significant levels of indentation.